source/world_builder/features/subducting_plate_models/temperature/mass_conserving.cc

```diff
@@ -121,7 +121,7 @@ namespace WorldBuilder
           prm.declare_entry("plate velocity", Types::OneOf(Types::Double(0.05),Types::Array(Types::ValueAtPoints(0.05, std::numeric_limits<uint64_t>::max()))),
                             "The velocity with which the plate subducts in meters per year. Default is 5 cm/yr");
 
-          prm.declare_entry("subducting velocity", Types::OneOf(Types::Double(-1), Types::Array(Types::Array(Types::Double(-1), 1), 1)),
+          prm.declare_entry("subducting velocity", Types::OneOf(Types::Double(0.05), Types::Array(Types::Array(Types::Double(0.05), 1), 1)),
                             "The velocity with which the ridge is moving through time, and how long the ridge "
                             "has been moving. First value is the velocity, second is the time. Default is [0 cm/yr, 0 yr]");
```

Comment:
Since you're already changing stuff in this file can you please change the description of subducting velocity? It should instead be:

 "The velocity with which the slab is subducting in meters per year. Default is 5 cm/yr".